pageserver/src/bin/pageserver.rs

```diff
@@ -501,6 +502,12 @@ fn start_pageserver(
     // Set up remote storage client
     let remote_storage = BACKGROUND_RUNTIME.block_on(create_remote_storage_client(conf))?;
 
+    let feature_resolver = BACKGROUND_RUNTIME.block_on(create_feature_resolver(
```

Comment:
`BACKGROUND_RUNTIME.block_on` is not needed anymore, because `create_feature_resolver` is not required to be called from tokio runtime